src/main/java/use_cases/custom_game/custom_game_file_management/CustomGameValidator.java

```diff
@@ -26,11 +30,18 @@ public static boolean verifyMaze(EditorTile[][] maze){
 
     /**
      * Checks if a maze name is unique
+     *
      * @param name the maze name to check
      * @return whether the name is unique or not
      */
     public static boolean verifyName(String name){
-        //TODO - when the initializer is made
-        return true;
+        File[] mazeFileList = new File("customMazes/").listFiles();
+        ArrayList<String> mazeList = new ArrayList<>();
+
+        assert mazeFileList != null;
```

Comment:
It might be better to do something like this instead:
```java
if (mazeFileList == null)
    throw new RuntimeException("customMazes folder does not exist.");
```
but it's not really a big deal since that folder should always exist in theory...

Reply:
I will add it to the next pull request, thank you